docs/api.md

```diff
@@ -452,10 +453,15 @@ The default flags that Chromium will be launched with.
   - `pipe` <[boolean]> Connects to the browser over a pipe instead of a WebSocket. Defaults to `false`.
 - returns: <[Promise]<[Browser]>> Promise which resolves to browser instance.
 
-The method combines 3 steps:
+The method combines 4 steps:
 1. Infer a set of flags to launch Chromium with using `puppeteer.defaultArgs()`.
-2. Launch a browser and start managing its process according to `executablePath`, `handleSIGINT`, `dumpio` and other options.
-3. Create an instance of [Browser] class and initialize it with regards to `defaultViewport`, `slowMo` and `ignoreHTTPSErrors`.
+2. Infer `executablePath` to use. Executable path is inferred according to the following priorities, from greatest to least:
+  * use `executablePath` option if set.
```

Comment:
This sublist is now aligned with the numbered items. Maybe add 2 spaces to each item in the sublist?

Reply:
Dropped the whole thing.